.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+<!--  Thanks for sending a pull request!  Here are some tips for you:
+1. If you want *faster* PR reviews, read how: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md
```

Comment:
This should link to the Helm contributor guide

Reply:
Done